bittensor/_neuron/text/core_validator/__init__.py

```diff
@@ -234,6 +234,7 @@ def add_args( cls, parser ):
         parser.add_argument('--neuron.print_neuron_stats', action='store_true', help='If True, print neuron_stats and exit.', default=False)
         parser.add_argument('--neuron.restart', action='store_true', help='If True, reset neuron_stats and validate anew.', default=False)
         parser.add_argument('--neuron.restart_on_failure',  action='store_true', help='''Restart neuron on unknown error.''', default=True )
+        parser.add_argument('--neuron.no_serve',  action='store_true', help='If True, neuron wont serve axon (disables prometheus)', default=True )
```

Comment:
This needs to default to `False`, or else, the store_action would not do anything